weavelet.go

```diff
@@ -455,19 +455,18 @@ func (w *weavelet) UpdateComponents(req *protos.UpdateComponentsRequest) (*proto
 
 // UpdateRoutingInfo implements the conn.WeaverHandler interface.
 func (w *weavelet) UpdateRoutingInfo(req *protos.UpdateRoutingInfoRequest) (reply *protos.UpdateRoutingInfoReply, err error) {
-	// TODO(rgrandl): After we switch to slog, call With here to avoid
-	// repeating the same attributes again and again.
-	attrs := []any{
+	logger := w.env.SystemLogger().With([]any{
 		"component", req.RoutingInfo.Component,
 		"local", req.RoutingInfo.Local,
 		"replicas", req.RoutingInfo.Replicas,
-	}
-	w.env.SystemLogger().Debug("Updating routing info...", attrs...)
+	}...)
```

Comment:
Can we remote the `[]any` part and just do the following?

```go
logger := w.env.SystemLogger().With(
	"component", req.RoutingInfo.Component,
	"local", req.RoutingInfo.Local,
	"replicas", req.RoutingInfo.Replicas,
)
```

Reply:
Good point.